lib/cfme-versions.rb

```diff
@@ -91,7 +91,7 @@ def [](index)
 
       # Version of this gem/tool
       def version
-        versions.last.cfme_release.split(".").select { |val| val =~ /^\d+$/ }.join('.')
+        versions.last.cfme_release.split(".").select { |val| val =~ /^\d+$/ }.join('.') + "-1"
```

Comment:
Will this work with releasing a gem?  I'm not sure rubygems will accept it?  (I was thinking `5.12.1`, FWIW)

Reply:
See https://github.com/RedHatCloudForms/cfme-versions/pull/7#discussion_r326732505

But yeah, wasn't sure if it would work, just didn't want it to be confusing to the user with a `.1` thinking it was associated to a `z-stream` release.  Either really works though.

Reply:
Also... does this project now need a `CHANGELOG.md`.... lol

Reply:
@Fryguy okay, updated with just a standard patch version release.  Included a `README.md` explanation as well for future reference.